src/utils/get-unique-tags.js

```diff
@@ -14,7 +14,7 @@ module.exports = collection => {
 	collection.forEach(item => {
 		if (!item.data.tags) {return;}
 		item.data.tags
-			.filter(tag => !["pages", "initiatives", "news", "views", "comments"].includes(tag))
+			.filter(tag => !["pages", "initiatives", "news", "events", "comments"].includes(tag))
```

Comment:
do we need to update news to recount here?

Reply:
Maybe, let me check!